pdfannots/printer/json.py

```diff
@@ -15,28 +15,19 @@ def annot_to_dict(
 
     result = {
         "type": annot.subtype.name,
-        "page": annot.pos.page.pageno + 1,
+        "page_number" "number": annot.pos.page.pageno + 1,
+        "page_label": getattr(annot.pos.page, 'label', None),
```

Comment:
I don't understand why you need to use getattr here (nor on the subsequent lines). Every page object has a label field. That field might be None, which you handle below on line 30, but the field always exists.

Reply:
OK, I removed it now for all the `annot` instances. I kept it in for the `prior_outline` as I am not sure whether that is also always set. But feel free to remove it there as well.

Reply:
That's the one place you need it